scripts/run-integration-tests.sh

```diff
@@ -15,8 +15,8 @@ TESTS_TO_RUN="${TESTS_TO_RUN:-all}"
 
 if [[ "$TESTS_TO_RUN" == "all" ]]; then
   echo "run all tests"
-  pytest -vv -s
+  pytest -vv -s --timeout=3600
```

Comment:
the timeout value is for each test case, not the whole run, right?

Reply:
yea, it's global one, do you mean we add pytest.mark.timeout for each test?

Reply:
I mean the 1hour limit is put on a individual test case, or the whole run.

Reply:
1h is for whole run, but we can overwrite in each test with shorter timeout

Reply:
> 1h is for whole run, but we can overwrite in each test with shorter timeout

I think the `--session-timeout` is for whole run, `timeout` is for a test case.